locale/Makefile

```
@@ -32,9 +32,9 @@ $(foreach lang,$(WX_LINGUAS_UPDATE),$(lang).po): wxstd.pot
 
 wxstd.pot:
 	touch $@
-	find ../include -name "*.h" | $(XARGS) $(XGETTEXT) $(XGETTEXT_ARGS) -o wxstd.pot
-	find ../src -name "*.cpp" | $(XARGS) $(XGETTEXT) $(XGETTEXT_ARGS) -o wxstd.pot
-	find ../src -name "*.mm" | $(XARGS) $(XGETTEXT) $(XGETTEXT_ARGS) -o wxstd.pot
+	find ../include -name "*.h" | LC_COLLATE=C sort | $(XARGS) $(XGETTEXT) $(XGETTEXT_ARGS) -o wxstd.pot
+	find ../src -name "*.cpp" | LC_COLLATE=C sort | $(XARGS) $(XGETTEXT) $(XGETTEXT_ARGS) -o wxstd.pot
+	find ../src -name "*.mm" | LC_COLLATE=C sort | $(XARGS) $(XGETTEXT) $(XGETTEXT_ARGS) -o wxstd.pot
```

Do you think it could be worth it to also replace these pipelines with a single one, i.e.
```suggestion
	(find ../include -name "*.h"; find ../src -name "*.cpp"; find ../src -name "*.mm") | LC_COLLATE=C sort | $(XARGS) $(XGETTEXT) $(XGETTEXT_ARGS) -o wxstd.pot
```
?
(the last 2 find commands could also be combined using `-o`, but this is even more minor).

Also, just out of curiosity, why do we need to set `LC_COLLATE` explicitly? All our filenames are ASCII, so sorting should work the same in all locales, AFAIK.

---

> Do you think it could be worth it to also replace these pipelines with a single one

Good idea. I squashed that to my commit.

> Also, just out of curiosity, why do we need to set `LC_COLLATE` explicitly? All our filenames are ASCII, so sorting should work the same in all locales, AFAIK.

That was my initial intuition as well, but turns out that `src/gtk/xxx.cpp` and `src/gtk1/xxx.cpp` are ordered differently in `C` and `fi_FI`. Also many locales seem to sort lowercase letters before uppercase, unlike `C` does.

---

> > Also, just out of curiosity, why do we need to set `LC_COLLATE` explicitly? All our filenames are ASCII, so sorting should work the same in all locales, AFAIK.
> 
> That was my initial intuition as well, but turns out that `src/gtk/xxx.cpp` and `src/gtk1/xxx.cpp` are ordered differently in `C` and `fi_FI`. Also many locales seem to sort lowercase letters before uppercase, unlike `C` does.

Thanks, good to know (I should have thought about case sensitivity but I had absolutely no idea about the Finnish locale, so I learnt something new today)!